source/conf.py

```diff
@@ -1508,7 +1508,7 @@ def setup(_: Sphinx):
 "configure/configuration-settings.html#user-status-away-timeout":
         "https://docs.mattermost.com/configure/experimental-configuration-settings.html#user-status-away-timeout",
 "configure/configuration-settings.html#enable-shared-channels":
-        "https://docs.mattermost.com/configure/experimental-configuration-settings.html#enable-shared-channels",
+        "https://docs.mattermost.com/configure/site-configuration-settings.html#enable-shared-channels",
```

Comment:
Should this anchor still be called "enable-shared-channels" ?

Reply:
Nice catch, @wiggin77! Thank you!

Reply:
There are a few instances of this.

Reply:
I believe all the remaining instances left are for the "from" portion of each page redirect, and not the defined destination. Are you seeing differently?